hack/ci/e2e-k8s.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
this was intentionally written in posix sh since the logic is simple.

Reply:
Switched in order to source hack/lib/util.sh. Thought process: bash is fairly universal nowadays, used in most places in k/*, and all other shell scripts in this directory use bash. It can still be simple even if it's bash :)